cylc/flow/scripts/tui.py

```diff
@@ -55,57 +58,36 @@
 def get_option_parser() -> COP:
     parser = COP(
         __doc__,
-        argdoc=[WORKFLOW_ID_ARG_DOC],
+        argdoc=[OPT_WORKFLOW_ID_ARG_DOC],
         # auto_add=False,  NOTE: at present auto_add can not be turned off
         color=False
     )
 
-    parser.add_option(
-        '--display',
```

Comment:
Removed the HTML testing mode from the Tui script as this is now handled by an integration test fixture.